Update scripts to account for new monorepo location

caseydavenport opened the pull request.

caseydavenport committed:
Update scripts to account for new monorepo location

caseydavenport committed:
Fix location of rpm script within container

caseydavenport committed:
Temporarily comment out semaphore prev-release build

nelljerram commented on utils/make-packages.sh:
```diff
@@ -8,6 +8,9 @@
 scriptdir=$(dirname $(realpath $0))
 
+# Get the location of the rpm builder script.
+rpmDir=$(dirname $(realpath $scriptdir))/rpm
```
$scriptdir is already canonical, so this can be simplified to just `rpmDir=$(dirname $scriptdir)/rpm`

nelljerram commented on .semaphore/semaphore.yml:
```diff
@@ -42,10 +42,12 @@ blocks:
+            - export VERSION=release-v3.21
             - make release-publish
+        # TODO: Uncomment this once v3.22 is released. v3.20 is not monorepo compatible,
+        # so needs to use an older version of the packagining repository.
```
sp: packaging

nelljerram commented:
@caseydavenport As a further way forward, I would suggest:

- pull the content of this repo into the monorepo
- use the in-monorepo packaging code to do packaging for the monorepo
- use the legacy packaging code to do any legacy packaging that we need for older branches.

caseydavenport committed:
Fix publishing by moving files to root

Commit:
Remove Makefile from Felix source package so DH doesn't try to use it

If "Makefile" exists, dh_auto_install (and other dh programs) will run
"make" to try to discover if it has an "install" target, and then we
hit a problem with the `include ../metadata.mk`:

     fakeroot debian/rules binary
    dh binary  --with systemd
       dh_testroot
       dh_prep
       dh_auto_install
    	make -j1 install DESTDIR=/<<PKGBUILDDIR>>/debian/tmp AM_UPDATE_INFO_DIR=no
    make[1]: Entering directory '/<<PKGBUILDDIR>>'
    Makefile:1: ../metadata.mk: No such file or directory
    Makefile:60: ../lib.Makefile: No such file or directory
    make[1]: *** No rule to make target '../lib.Makefile'.  Stop.
    make[1]: Leaving directory '/<<PKGBUILDDIR>>'
    dh_auto_install: make -j1 install DESTDIR=/<<PKGBUILDDIR>>/debian/tmp AM_UPDATE_INFO_DIR=no returned exit code 2
    debian/rules:7: recipe for target 'binary' failed

In fact we don't want the build on Launchpad to access Makefile at
all, and it seems to work if we simply delete it.

caseydavenport committed:
Fix upload of RPMs

caseydavenport committed:
Fix upload of RPMs

Commit:
Comment building of old Python 2 branch
